frontend/language/src/nb.json

```
@@ -655,6 +655,8 @@
   "process_editor.not_found_process_error_message": "Det finnes ingen prosess definert innenfor BPMN, Du kan verifisere om prosessen finnes i BPMN-filen.",
   "process_editor.not_found_process_heading": "Ingen tilgjengelig prosess",
   "process_editor.save": "Lagre",
+  "process_editor.too_old_version_text": "Applikasjonen din har versjon {{ versionNo }} av app-lib-pakken.\nVersjonen er for gammel for å kunne redigere denne modellen.\n\nDu kan se på prosessen, men vi anbefaler at du oppgraderer til versjon 8 eller nyere.\n\nTrenger du hjelp til å oppgradere, kan du kontakte <0>servicedesken</0> vår.",
```

Ser at denne teksten kommer fra figma-designet. Jeg ville kanskje foreslått noe mindre "hardt", i og med at alle apper som opprettes nå fortsatt er "for gamle" til å støtte ny prosessEditor. 

Tenker at vi også:
1. Bør lenke til vår dokumentasjon rundt hvordan man oppgraderer version: https://docs.altinn.studio/nb/app/maintainance/dependencies/
2. I stedet for å be folk om å kontakte servicedesk, lenke til vår kontakt-side.

Noe ala:

```suggestion
  "process_editor.too_old_version_text": "Applikasjonen din har versjon {{ versionNo }} av app-lib-pakken.\nVi støtter kun redigering av prosesser for applikasjoner som bruker versjon 8 eller nyere.\n\nDu kan se på prosessen, men for å redigere den i dette verktøyet må du oppgradere til versjon 8 eller nyere.\n\nLes mer om hvordan dette gjøres <0>her</0> Trenger du hjelp til å oppgradere, kan du <0>kontakte oss</0>.",
```

@Annikenkbrathen tanker?

---

Jeg er enig i det du sier Nina. 😊

---

Enig! Fint å bruke kontaktsiden mest mulig ja